mypy/checkexpr.py

```diff
@@ -3318,7 +3333,7 @@ def visit_ellipsis(self, e: EllipsisExpr) -> Type:
         """Type check '...'."""
         return self.named_type("builtins.ellipsis")
 
-    def visit_op_expr(self, e: OpExpr) -> Type:
+    def visit_op_expr(self, e: OpExpr, *, use_reverse: UseReverse = USE_REVERSE_DEFAULT) -> Type:
```

Comment:
I was thinking about having it as an argument in `check_op` _instead_ of `allow_reverse`, but if it makes more sense here, then it should be a local variable. I don't see any caller passing in `use_reverse`.